### DISCUSSION TITLE
📺 普林斯顿 COS226 算法和数据结构 ⭐️ | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#52] [Announcements]
# 📺 普林斯顿 COS226 算法和数据结构 ⭐️ | HackWay技术学习路线

普林斯顿算法和数据结构 Princeton COS226 Algorithms and Data Structures

https://hackway.org/docs/cs/freshman/datastructure/cos226

**Comment by KingofKanya** [comment]
这个不适合毫无基础的

**Reply by hackwayorg** [reply]
不是有先修课程嘛
